drivers/ethernet/eth_stm32_hal.c

```diff
@@ -60,18 +175,31 @@ static void set_mac_to_translator_addr(uint8_t *mac_addr)
 static void enable_canbus_eth_translator_filter(ETH_HandleTypeDef *heth,
 						uint8_t *mac_addr)
 {
+	/* @TODO: must be done for STM32H7 series also! */
 	heth->Instance->MACA1LR = (mac_addr[3] << 24U) | (mac_addr[2] << 16U) |
 				  (mac_addr[1] << 8U) | mac_addr[0];
-	/*enable filter 1 and ignore byte 5 and 6 for filtering*/
+	/* enable filter 1 and ignore byte 5 and 6 for filtering */
 	heth->Instance->MACA1HR = ETH_MACA1HR_AE |  ETH_MACA1HR_MBC_HBits15_8 |
 				  ETH_MACA1HR_MBC_HBits7_0;
 }
-#endif /*CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR*/
+#endif /* CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR */
```

Comment:
Please perform clean up in a dedicated commit

Reply:
done